sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1534,6 +1550,10 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setIgnoreUnknownValues(true).build();
     }
 
+    Write<T> withKmsKey(String kmsKey) {
```

Comment:
Shouldn't this method be public?

Reply:
Yes, it should be.

Reply:
Can we reopen this as a hotfix, or should I create a new issue to track this? As far as I can tell there's no way to actually use this feature.

Reply:
If you want a patch to be applied to the current LTS family (version 2.7) then please open a new ticket for that.

Reply:
@mayansalama I fixed this here: https://github.com/apache/beam/pull/8145. I should have updated you.
@kennknowles KMS support is too new to be in 2.7. :)